apps/fluent-tester/src/RNTester/TestComponents/FocusZone/FocusZoneTest.tsx

```diff
@@ -91,45 +47,99 @@ const commonUsageFocusZone: React.FunctionComponent<{}> = () => {
+          <RadioButton content="Option B" buttonKey="B" />
+          <RadioButton content="Option C" buttonKey="C" disabled={true} />
+          <RadioButton content="Option D" buttonKey="D" />
+        </RadioGroup>
+      </FocusZone>
+      <Button content="Outside FocusZone" />
     </View>
   )
 };
 
-const directionalFocusZone: React.FunctionComponent<{}> = () => {
+const commonUsageFocusZone: React.FunctionComponent<{}> = () => {
   return (
     <View>
       <Button content="Outside FocusZone" />
-      <FocusZone disabled={null} focusZoneDirection='bidirectional'>
-        <RadioGroup label="FocusZone RadioGroup with Bidirectional arrow key navigation" defaultSelectedKey="A">
+      <RadioGroup label="FocusZone RadioGroup with Circular Navigation" defaultSelectedKey="A">
+        <FocusZone isCircularNavigation={true}>
           <RadioButton content="Option A" buttonKey="A" ariaLabel="Cool" />
           <RadioButton content="Option B" buttonKey="B" />
           <RadioButton content="Option C" buttonKey="C" disabled={true} />
           <RadioButton content="Option D" buttonKey="D" />
-        </RadioGroup>
-      </FocusZone>
+        </FocusZone>
+      </RadioGroup>
       <Button content="Outside FocusZone" />
-      <FocusZone focusZoneDirection='vertical'>
-        <RadioGroup label="FocusZone RadioGroup with Vertical arrow key navigation" defaultSelectedKey="A">
+      <FocusZone isCircularNavigation={false}>
+        <RadioGroup label="FocusZone RadioGroup with no Circular Navigation" defaultSelectedKey="A">
           <RadioButton content="Option A" buttonKey="A" ariaLabel="Cool" />
           <RadioButton content="Option B" buttonKey="B" />
           <RadioButton content="Option C" buttonKey="C" disabled={true} />
           <RadioButton content="Option D" buttonKey="D" />
         </RadioGroup>
       </FocusZone>
       <Button content="Outside FocusZone" />
-      <FocusZone focusZoneDirection='horizontal'>
-        <RadioGroup label="FocusZone RadioGroup with Horizontal arrow key navigation" defaultSelectedKey="A">
+      <FocusZone disabled={true}>
+        <RadioGroup label="Disabled" defaultSelectedKey="A">
           <RadioButton content="Option A" buttonKey="A" ariaLabel="Cool" />
           <RadioButton content="Option B" buttonKey="B" />
           <RadioButton content="Option C" buttonKey="C" disabled={true} />
           <RadioButton content="Option D" buttonKey="D" />
         </RadioGroup>
       </FocusZone>
       <Button content="Outside FocusZone" />
+      <FocusZone disabled={true}>
+        <Checkbox label="disabled" />
+        <Checkbox label="disabled" />
+        <Checkbox label="disabled" />
+      </FocusZone>
+      <Button content="Outside FocusZone" />
     </View>
   )
 };
 
+
+const navigation2DFocusZone: React.FunctionComponent<{}> = () => {
+  const buttonRef = React.useRef<View>(null);
+
+  // const onFocus = () => {
```

Comment:
nit: is this needed?

Reply:
Not anymore, I was using it to test with. I'll remove it

Reply:
Removed in newest commit